src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.php

```diff
@@ -151,6 +152,9 @@
         ->set('translation.extractor.php', PhpExtractor::class)
             ->tag('translation.extractor', ['alias' => 'php'])
 
+        ->set('translation.extractor.php_ast', PhpAstExtractor::class)
+            ->tag('translation.extractor', ['alias' => 'php_ast'])
```

Comment:
this should use the `php` alias as well, it is is meant to replace the PhpExtractor to extract translations from `php` files

Reply:
it means we will deprecate `PhpExtractor` in 6.2 and remove it in 7.0, so making `niki/php-parser` mandatory to use PHP file extraction for translation in 7.0?

Reply:
no. But the DI extension already removes one of these 2 services all the time. And with a `php_ast` alias, it would apply only on files with a `.php_ast` extension.